docs/source/membership/membership.md

```diff
@@ -236,106 +376,75 @@ Let's describe these folders in a little more detail and see why they are import
   Like the Root CA folder, this folder defines the CAs from which certificates must be
   issued to be considered members of the organization.
 
+* **tlscacerts:** This folder contains a list of self-signed X.509 certificates of the
+  Root CAs trusted by this organization **for TLS communications**. An example of a TLS
```

Comment:
There isn't TLS communication, right? It's not a separate layer of communication (I don't think). It's just the certificates that go along with communication to secure it.

Reply:
I changed it to **for secure communications between nodes using TLS**